idd/Energy+.idd.in

```diff
@@ -99616,6 +99639,7 @@ Output:Meter,
        \note RunPeriod, Environment, and Annual are the same
        \key Hourly
        \key Daily
+       \key Detailed
```

Comment:
A quick inspection of the source code says that all eight keys are accepted for meters.  So, letting Detailed stay here and adding to other meter output objects.